lib/katello/plugin.rb

```diff
@@ -171,4 +171,11 @@
                    :partial => 'smart_proxies/show/storage',
                    :onlyif => proc { |proxy| proxy.has_feature?(SmartProxy::PULP_FEATURE) || proxy.has_feature?(SmartProxy::PULP_NODE_FEATURE) }
   end
+
+  RemoteExecutionFeature.register(:katello_package_install, N_("Katello: Install Package"), :description => N_("Install package via Katello interface"), :provided_inputs => ['package'])
```

Comment:
Do these need to be prepended with 'Katello' ? I am not seeing the value. Can you explain what this means by installing via the Katello interface?